src/lib/experimental/canister_methods/methods/update.ts

```diff
@@ -18,8 +18,7 @@ export function update<
     callback?: Awaited<ReturnType<GenericCallback>> extends TypeMapping<Return>
         ? GenericCallback
         : never,
-    methodArgs?: MethodArgs,
-    noop?: boolean
+    methodArgs?: MethodArgs
```

Comment:
What was this noop and why did we have it, and why are we able to get rid of it now

Reply:
Oh I see, we needed it for the type tests to avoid this very issue, but now that it is resolved we no longer need it